Orange/projection/manifold.py

```diff
@@ -109,7 +109,7 @@ class MDS(SklProjector):
     __wraps__ = skl_manifold.MDS
     name = 'MDS'
 
-    def __init__(self, n_components=2, metric=True, n_init=4, max_iter=300,
+    def __init__(self, n_components=2, metric=True, n_init=None, max_iter=300,
```

Comment:
I would prefer if our wrappers retain the same signature/defaults as the wrapped function as much as possible.
It is then much easier to update them, if/when these values change upstream (and they often do).

This leaves us with a couple of choices of how to combat the current awkward defaults:
- Fix the calls of MDS, and explicitly set `n_init` to 1 when `init` is used. Not sure it that is how the authors imagined it... (does not sound appealing)
- Let sklearn change it and filter out the warning when calling fit, if that bothers us
- We make the same change before them in our call. Basically what you did in L150, but do it when `init` is given instead of when `n_init` is not.

Reply:
The first option was the simplest because this happened only in one place.